server/services/identity_service_test.go

```diff
@@ -33,7 +33,7 @@ func Test_User_lifecycle(t *testing.T) {
 	first, err := s.CreateUser(
 		context.Background(),
 		&pb.CreateUserRequest{
-			User: &pb.User{DisplayName: "ekkodog", Email: "ekko@google.com"},
+			User: &pb.User{DisplayName: "ekkodog", Email: "ekko@google.com", Nickname: proto.String("ekko"), Age: proto.Int32(26)},
```

Comment:
It might be a good time to start using `@example.com` instead of `@google.com` emails (see [here](https://www.iana.org/domains/reserved) for more information).

Reply:
I opened a cleanup task to refactor all uses of `google.com` to `example.com`: #381. There is another such cleanup task we will need to get to as well.